Source/SwiftLintBuiltInRules/Rules/Lint/UnhandledTryInTaskRule.swift

```diff
@@ -0,0 +1,170 @@
+import SwiftSyntax
+
+struct UnhandledTryInTaskRule: ConfigurationProviderRule, SwiftSyntaxRule {
+    var configuration = SeverityConfiguration(.error)
+
+    static let description = RuleDescription(
+        identifier: "unhandled_try_in_task",
+        name: "Unhandled Try in Task",
```

Comment:
Since there might not be a `try` keyword in the body of the task, this might be misleading.

Perhaps the rule should be `unhandled_throwing_task`?

```suggestion
        identifier: "unhandled_throwing_task",
        name: "Unhandled Throwing Task",
```